src/models/Players.js

```diff
@@ -1,27 +1,27 @@
-const mongoose = require("mongoose")
+const mongoose = require("mongoose");
 
 const PlayerSchema = mongoose.Schema({
-    username: {
-        type: String,
-        required: true,
-    },
-    level: {
-        type: String,
-        required: false,
-    },
-    score: {
-        type: String,
-        required: false,
-    },
-    email: {
-        type: String,
-        required: false,
-        default: 'email@email.com'
-    },
-    group_id: {
-        type: String,
-        required: false,
-    },
-})
+  username: {
+    type: String,
+    required: true,
+  },
+  level: {
+    type: String,
+    required: false,
+  },
+  score: {
+    type: String,
+    required: false,
+  },
+  email: {
+    type: String,
+    required: false,
+    default: "email@email.com",
+  },
+  groupId: {
```

Comment:
- this'll need to be updated in Swagger documentation, too. I'll handle this
- @jfarelli this also changed; so heads up if it's being used in the front end